sw.cpp

```diff
@@ -21,7 +21,7 @@ void build(Solution &s)
 
         libtesseract -=
             "src/tesseract.cpp",
```

Comment:
What about this line?

Reply:
Maybe the lines 22...24 are no longer needed at all and can be removed.